content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>Suggestion:</p><p> </p><ul><li>Capitalize Matplotlib</li><li>nit: Instead of "read medical images" (which might have a connotation of interpreting them) I'd suggest using "load medical images". But I'm not even sure if that sounds the same to a native english speaker.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done! 👍